proposals/NNNN-set-language-separately.md

```diff
@@ -0,0 +1,55 @@
+# Set language separately

```

Comment:
This should be updated to `Support fo
```suggestion
# Support for Set Language Separately
```

Please make the title of your pull request the exact title of your proposal.